public/locales/en/translation.json

```diff
@@ -1,20 +1,19 @@
 {
-  "title": "Project Title",
-  "nav": {
-    "dropdownOne": {
-      "label": "Nav Label 1",
-      "simpleLinkOne": "Simple link one",
-      "simpleLinkTwo": "Simple link two"
+  "header": {
```

Comment:
This translation file is still in `public`. I wonder if it would make more sense at this point, since this PR is so old, to create a new PR off of dev once we merge language switcher. Then we can update all 13 translation files, which are now in `src`.  We can also copy and paste your header and footer changes into the new branch?

Reply:
Sounds good.